### PR TITLE
[Antic_jll] yank bad version

### PR DESCRIPTION
**Pull request by thofma**
We made an oopsie in https://github.com/JuliaPackaging/Yggdrasil/pull/2264.

### CODE REVIEW

File: N/Nemo/Compat.toml [hunk 1]
```diff
@@ -65,7 +65,7 @@ Arb_jll = "2.17.0"
 LoadFlint = "0.2.1-0.2"
 
 ["0.18-0"]
-Antic_jll = "0.2.2-0.2"
+Antic_jll = "0.2.2-0.2.2"
```
Comment (giordano) [inline N/Nemo/Compat.toml:68]: Why do you need to change this if the broken version is yanked?

Comment (thofma) [reply N/Nemo/Compat.toml:68]: julia 1.0

Comment (giordano) [reply N/Nemo/Compat.toml:68]: Julia v1.0 wouldn't use the JLL anyway

Comment (thofma) [reply N/Nemo/Compat.toml:68]: I don't like that it will pull in 0.2.3 on julia 1.0 and 0.2.2 on julia >= 1.1. I don't like the inconsistent state.

**Comment by thofma** [comment]
Thanks!

File: N/Nemo/Compat.toml (hunk 1 above)
Comment (fredrikekre) [reply N/Nemo/Compat.toml:68]: That seems like a strange argument, basically no package install the same version on all Julia versions :shrug:

Comment (thofma) [reply N/Nemo/Compat.toml:68]: Which package with no restriction on the julia version in the `[compat]` section installs different versions on different julia versions?

Comment (thofma) [reply N/Nemo/Compat.toml:68]: Which package with no restriction on the julia version in the `[compat]` section installs different versions on different julia versions?

**Pull request merged**